BitwardenShared/Core/Platform/Services/Stores/AppSettingsStore.swift

```diff
@@ -21,6 +21,11 @@ protocol AppSettingsStore: AnyObject {
     /// The app's theme.
     var appTheme: String? { get set }
 
+    /// The last published active user ID by `activeAccountIdPublisher` in the current process.
+    /// If this is different than the active user ID in the `State`, the active user was likely
+    /// switched in an extension and the main app should update accordingly.
+    var cachedActiveUserId: String? { get }
```

Comment:
🤔 is it possible for the extension to update the account on disk vs checking in the foreground and doing it within the main app?

Reply:
The extension does update the active user on disk, but if the main app is still in memory it will continue to show the data that's in memory for the previous active user when you return to it. The check I've added here causes the app to update once it's been foregrounded if the active user (on disk) has changed.